wdl-grammar/tests/parsing/precedence/source.wdl

```diff
@@ -0,0 +1,8 @@
+# This is a test of operator precedence
```

Comment:
So in the Rust code I saw assignment and calculation of precedence. But am I correct that it isn't tested or reflected at this stage of the parse implementation? `source.tree` is just in the order it was found in the file, and doesn't have precedence information.

Reply:
The precedence is captured in the depth of the tree, though. It just happens that the expressions are in lowest-to-highest order, so the tree shape matches.

I'll add a reverse test.